lib/util/git/index.ts

```diff
@@ -900,7 +903,30 @@ export async function mergeBranch(branchName: string): Promise<void> {
       ]),
     );
     status = await git.status();
-    await gitRetry(() => git.merge(['--ff-only', branchName]));
+    if (
+      mergeStrategy === 'fast-forward' ||
+      mergeStrategy === 'rebase' ||
+      mergeStrategy === 'auto'
+    ) {
+      await gitRetry(() => git.merge(['--ff-only', branchName]));
+    } else if (mergeStrategy === 'merge-commit') {
+      await gitRetry(() =>
+        git.merge([
+          '--no-ff',
+          '--no-edit',
+          '-m',
+          'Automerge branch ' + branchName + ' by Renovate Bot',
```

Comment:
This is ignoring the user's settings for semantic commits. If it can't be easily supported, then it needs to be documented as a limitation. Same for squash below

Reply:
I've added the `automergeMergeCommitMessage`. If set, this message will be used, else the default merge messages from git itself as before.